class.jetpack-gutenberg.php

```diff
@@ -689,4 +689,46 @@ public static function load_independent_blocks() {
 			}
 		}
 	}
+
+	/**
+	 * Get CSS classes for a block.
+	 *
+	 * @since 7.7.0
+	 *
+	 * @param string $slug  Block slug.
+	 * @param array  $attr  Array containing the block attributes.
+	 * @param array  $extra Array of potential extra classes you may want to provide.
+	 *
+	 * @return string $classes List of CSS classes for a block.
+	 */
+	public static function block_classes( $slug = '', $attr, $extra = array() ) {
+		if ( empty( $slug ) ) {
+			return '';
+		}
+
+		// Basic block name class.
+		$classes = array(
+			'wp-block-jetpack-' . $slug,
+		);
+
+		// Add alignment if provided.
+		if (
+			! empty( $attr['align'] )
+			&& in_array( $attr['align'], array( 'left', 'center', 'right', 'wide', 'full' ), true )
+		) {
+			array_push( $classes, 'align' . $attr['align'] );
+		}
```

Comment:
Do we even have to take care of alignment? I thought we could pretty much rely on Gutenberg's [`supports`](https://developer.wordpress.org/block-editor/developers/block-api/block-registration/#supports-optional) feature for alignment... :thinking: I'll try to dig up a PR I vaguely recall where I was doing something like that...

Reply:
I think some of our blocks are little old in that we handled it manually. Gutenberg's support might be purely frontend side and backend side has to be handled manually like this?

Reply:
For reference, this is the commit I was thinking of: https://github.com/Automattic/wp-calypso/commit/d675ab08e826a646b05c0d9b78ed437f32290822

I'll see if that kind of change can be applied here...

Reply:
Ah, nevermind, that doesn't really apply to dynamic blocks/the server side/PHP.